src/image/components/app.tsx

```diff
@@ -17,8 +17,7 @@ export interface IAuthoredState {
   creditLinkDisplayText?: string;
   allowLightbox?: boolean;
   scaling?: "fitWidth" | "originalDimensions";
-  modalSupported?: boolean;
-  isShowingModal?: boolean;
+  isShowingModal?: boolean; // TODO: move this to initInteractive message
```

Comment:
So, I think I don't understand this line (also the previous ones). Why `isShowingModal` in authoredState?
Is ActivityPlayer injecting this value into authoredState when it's rendering interactive using modal?

Reply:
It's basically an ad hoc mechanism for Activity Player to communicate with the interactive. As the comment suggests, it would be better for there to be a better mechanism than using the authoredState for this purpose.